.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,236 @@
+
+        if isinstance(value, dict):
+            # Recursively flatten the nested dictionary
+            flat_dict.update(flatten_json(value, new_key))
+        else:
+            # Assign the value to the flattened key
+            flat_dict[new_key] = value
+
+    return flat_dict
+
+
+def load_translation(filepath):
```

Ruff (0.12.2)

Missing return type annotation for public function `load_translation`

(ANN201)